src/main_app/db/services/crud_service.py

```
@@ -116,20 +119,28 @@ def exists(self, **filters: Any) -> bool:
             self.commit()
+            self.session.refresh(instance)
             return instance
+        except Exception:
+            logger.error("Error adding %s", self.model_name)
+            raise
+
+    def create_safe(self, **fields: Any) -> ModelT | None:
+        """Instantiate the model with `fields` and persist it."""
+        try:
+            return self.create(**fields)
         except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)